api/spec/scanner-adapter-openapi-v1.1.yaml

```diff
@@ -0,0 +1,422 @@
+          example: "application/vnd.docker.distribution.manifest.v2+json"
+    HarborVulnerabilityReport:
+      type: object
+      properties:
+        generated_at:
+          type: string
+          format: 'date-time'
+        artifact:
+          $ref: '#/components/schemas/Artifact'
+        scanner:
+          $ref: '#/components/schemas/Scanner'
+        severity:
+          $ref: "#/components/schemas/Severity"
+        vulnerabilities:
+          type: array
+          items:
+            $ref: '#/components/schemas/VulnerabilityItem'
+    VulnerabilityItem:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The unique identifier of the vulnerability.
+          example: CVE-2017-8283
+        package:
+          type: string
+          description: |
+            An operating system package containing the vulnerability.
+          example: dpkg
+        version:
+          type: string
+          description: |
+            The version of the package containing the vulnerability.
+          example: 1.17.27
+        fix_version:
+          type: string
+          description: |
+            The version of the package containing the fix if available.
+          example: 1.18.0
+        severity:
+          $ref: "#/components/schemas/Severity"
+        description:
+          type: string
+          description: |
+            The detailed description of the vulnerability.
+          example: |
+            dpkg-source in dpkg 1.3.0 through 1.18.23 is able to use a non-GNU patch program
+            and does not offer a protection mechanism for blank-indented diff hunks, which
+            allows remote attackers to conduct directory traversal attacks via a crafted
+            Debian source package, as demonstrated by using of dpkg-source on NetBSD.
+        links:
+          type: array
+          items:
+            type: string
+            format: uri
+          description: |
+            The list of links to the upstream databases with the full description of the vulnerability.
+          format: uri
+          example:
+            - https://security-tracker.debian.org/tracker/CVE-2017-8283
+        cve3_score:
+          type: string
+          description: |
+            The CVSS 3.0 score for the vulnerability.
+          example: 3.2
+        cve2_score:
+          type: string
+          description: |
+            The CVSS 2.0 score for the vulnerability.
+        cvss3_vector:
+          type: string
+          description: |
+            The CVSS 3.0 vector for the vulnerability. The string is of the form CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N
+        cvss2_vector:
+          type: string
+          description: |
+            The CVSS 2.0 vector for the vulnerability. The string is of the form AV:L/AC:M/Au:N/C:P/I:N/A:N
+        vendor_attributes:
+          $ref: "#/components/schemas/VendorAttributes"
+    Severity:
```

Comment:
I'm wondering if it makes sense to tackle also the problem of using the standardized Severity levels for v2 and v3.
The Severity enum from version 1.0 of the API is inferred from Clair. However, I believe that we should be using CVSS v2 and / or CVSS v3 quantities

Reply:
+1